### PR TITLE
Update throughput measurements

### PR DESCRIPTION
**Pull request by hanlint**
This PR updates the throughput numbers in the new format for all meausrements except the 30B and 70B numbers on A100 80 GB systems.

### CODE REVIEW

**Comment by dakinggg** [comment]
Edited the base branch to be the release branch. These changes will get merged into main immediately after we release.

**Comment by abhi-mosaic** [comment]
Should we remove the `FULL_TABLES.md` file for now?

File: examples/llm/throughput/README.md [hunk 1]
```diff
@@ -63,8 +63,12 @@ python submit_benchmarks.py -m 13b.yaml 30b.yaml -t fp16 -b 21 21 -s 11 14 --RUN
 ```
 This will run 8 configs for 12 steps to get throughput numbers. `python collect_results.py` can then be used to parse all output training logs and create the tables below.
 
+Sometimes we report Global Batchsizes that are not evenly divisible by the micro batch size. Our microbatching engine allows for non-divisible sizes, while being mathematically faithful to the global batch size. For example, a total batch size of 48, and a micro batch of 11, means we will accumulate gradients across microbatches of 11, 11, 11, 11, 4.
```
Comment (dakinggg) [inline examples/llm/throughput/README.md:66]: Should the warning at the top of this file be removed yet?

Comment (abhi-mosaic) [reply examples/llm/throughput/README.md:66]: Probably not yet... the 13B/30B/70B models in some of the rows are still old. Maybe the warning should be reworded slightly though?

Comment (abhi-mosaic) [reply examples/llm/throughput/README.md:66]: Updated the warning a bit.

**Review by abhi-mosaic — APPROVED** [review]
LGTM

**Pull request merged**